add-ons/kyverno/values.yaml

```diff
@@ -0,0 +1,3 @@
+kyverno: 
+  hostNetwork: true
```

Comment:
HostNetwork is generally not a best pracice for running application. This is needed by Kyverno ?

Reply:
yes. Kyverno required.
https://github.com/aws-samples/eks-blueprints-add-ons/pull/44

Reply:
This seems only needed if using custom cni. As we recommend vpc cni we should point to the documentation in case user has trouble with it and not activate host network by default

Reply:
Is there some documentation where you can point us that shows, from AWS' side, that this is a requirement? We should update our docs to point back there.

Reply:
@allamand is correct -
> When using EKS with a custom CNI, the Kyverno webhook cannot be reached by the API server because the control plane nodes, which cannot use a custom CNI, differ from the configuration of the worker nodes, which can

Lets remove the `hostNetwork: true` please and we can merge this

Reply:
We're glad to do it, but once again, can you point us to some official documentation so we may steer our users to something which assists with their troubleshooting? Something which captures what @allamand said [here](https://github.com/aws-samples/eks-blueprints-add-ons/pull/44#discussion_r901280895)?

Reply:
I don't follow - the link to the kyverno documentation is provided above. what other documentation are you looking for?

Reply:
A link to AWS' documentation that states `hostNetwork: true` only when using a custom CNI plug-in.

Reply:
You won't find that in the EKS documentation because thats up to the provider of the CNI and what capabilities are required. This is all that EKS provides - a link to compatible CNIs and its up to those providers to list what is required https://docs.aws.amazon.com/eks/latest/userguide/alternate-cni-plugins.html . Some plugins, such as Calico with Calico networking, required `hostNetwork: true` and is listed in their documentation

Reply:
@bryantbiggs @allamand @chipzoller I have removed hostNetwork value